NEWS.md

```diff
@@ -5,7 +5,7 @@
 * Add support of link status and errorCause ([MODQM-318](https://issues.folio.org/browse/MODQM-318))
 
 ### APIs versions
-* Provides `marc-records-editor` `v5.2`
+* Provides `marc-records-editor` `v5.3`
```

Comment:
What about dependency to suggestion interface?

Reply:
Added